src/theme/_code.css

```diff
@@ -0,0 +1,13 @@
+.ck-content code {
+  border-radius: var(--ck-border-radius-small);
+  background-color: $ZD-PALETTE-GREY-200;
+  padding: 1.5px;
```

Comment:
`.ck-content pre code` should not be inheriting this padding.

Reply:
Updated to override padding.